hapi/src/api/account.api.js

```diff
@@ -57,7 +58,38 @@ const create = async ({ role, email, name, secret }) => {
 
   await historyApi.insert(transaction)
 
-  //mailApi.sendVerificationCode(email, verification_code)
+  mailApi.sendVerificationCode(email, verification_code)
```

Comment:
@AngeloCG97 what if we wrap this call on a try-catch statement?

Reply:
ok I'll do it now